pyproject.toml

```diff
@@ -75,3 +79,4 @@ repair-wheel-command = "auditwheel repair -w {dest_dir} {wheel} && python3 /proj
 
 [[tool.cibuildwheel.overrides]]
 select = "*-musllinux*"
+#TBD
```

Comment:
Elaborate, please.

Reply:
As you can see, it isn't new. It's a placeholder for musllinux specific options. I fixed musl builds a while back, but not passing the tests. If you want to remove the placeholder, we can.